### PR TITLE
Tech Fixes:

### PR DESCRIPTION
**Pull request by NanakoAC**
Fixes a number of issues with robot sprites, chem master machine, and surgery steps

The issue with borg sprites was caused by duplicate calls to ..()
Ive standardised robot module parent calls to the top of the function

Fixed the chemmaster not depositing pills in bottles properly. I added a new add_to_storage proc to storage.dm which is just a simple, safe means of putting stuff in stuff. it handles the checking.

Fixed some issues with surgery. A few surgery steps had incorrect "open" var ranges defined which was causing organ repair surgery to be interfered with

Also removed screwdriver from the list of bone mending tools, and added duct tape instead as the ghetto alternative. And allowed energyswords + chainswords to be used for cutting open ribs. theyre already useable for amputation

Fixed an incorrect return var for surgery that was making surgeons able to attack their patients on help intent. Shouldnt happen anymore

And also fixed the experimental welder being un-makeable at the protolathe.

### CODE REVIEW

File: code/modules/mob/living/silicon/robot/robot_modules.dm [hunk 1]
```diff
@@ -194,7 +195,7 @@ var/global/list/robot_modules = list(
 				"Heavy" = "heavyMed",
 				"Needles" = "medicalrobot",
 				"Standard" = "surgeon",
-				"Advanced Droid - Medical" = "droid-surgery",
+				"Advanced Droid - Medical" = "droid-medical",
```
Comment (NanakoAC) [inline code/modules/mob/living/silicon/robot/robot_modules.dm]: I think agent whatever caused this. 

I didnt see any others, hopefully it was an isolated mistake

**Comment by skull132** [comment]
Merge error from #1883

File: code/game/objects/items/weapons/storage/storage.dm [hunk 2]
```diff
@@ -413,7 +413,16 @@
 	update_icon()
 	return 1
 
+
 //This proc is called when you want to place an item into the storage item.
+//Its a safe proc for adding things to the storage that does the necessary checks. Object will not be moved if it fails
+/obj/item/weapon/storage/proc/add_to_storage(obj/item/W as obj, var/prevent_messages = 1)
```
Comment (skull132) [inline code/game/objects/items/weapons/storage/storage.dm]: I feel like it should have the word "insert" in the proc name. To fall in line with the other two procs.

**Comment by NanakoAC** [comment]
Merge issues addressed, skull's fix preserved, procname changed as requested

**Pull request merged**